201-managed-application-usingmsi-for-management/azuredeploy.json

```diff
@@ -0,0 +1,97 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "allowedValues": [ "australiaeast", "eastus", "westeurope" ],
```

Comment:
remove allowedValues

Reply:
Custom providers is only available in some regions I have to restrict this value, how can i do that ? 
we already do this for other custom provider samples. Check those here : 
https://github.com/Azure/azure-quickstart-templates/blob/master/101-custom-rp-with-function/azuredeploy.json

Reply:
ok, but please keep an eye on this, its a very small set of regions and I suspect will become stale quickly